generator.html

```diff
@@ -790,7 +790,8 @@ <h3>Formal Complaints</h3>
 
             <p>
                 This statement was created on
-                <span class="basic-information statement-created-date" data-print="accstmt_date" data-printdefault="DATE"></span>.
+                <span class="basic-information statement-created-date" data-print="accstmt_date" data-printdefault="DATE"></span>
+                using the <a href="{{ page.url | absolute_url }}">W3C Accessibility Statement Generator Tool</a>.
```

Comment:
@rvantonisse this should not be the URL of the generator page but of the parent directory (so that people don't land on the form but rather on the start page) - can you fix this please?

Reply:
Done! :)